### ISSUE TITLE
User Review

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/guide-jakarta-websocket#12]
User Review:

Allow a person with a fresh mind to read the guide and execute the steps described in the guide, test whether each step is clear enough to understand, and no logic errors.

Review and walkthrough by a non-SME or a new user with a critical and fresh pair of eyes:

- [x] Record the time it takes to go through the entire guide
- [x] Test each step in the guide is clear and easy to understand
- [x] Ensure no logic errors
- [x] Report anything that does not work
- [x] Report anything usability issues (confusing or hard to follow)
- [x] Does the guide achieve basic understanding/working knowledge of the topic in the guide?

**Comment by mbuamuh** [comment]
- It took 25 minutes to go through the guide as suggested by the author
- The guide is clear and easy to understand
- There are no logic errors
- The code works
- Its easy to follow
- Yes, the guide achieves basic understanding of its topic

**Comment by gkwan-ibm** [comment]
ref: https://github.com/OpenLiberty/guides-common/issues/34
